#! /bin/sh /usr/share/dpatch/dpatch-run
## xterm.dpatch by Francesco Paolo Lovergine <frankie@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: prefer to let alternatives chose the xterm used

@DPATCH@
diff -urNad grass-preview-6.3.0~/lib/init/grass-xterm-wrapper grass-preview-6.3.0/lib/init/grass-xterm-wrapper
--- grass-preview-6.3.0~/lib/init/grass-xterm-wrapper	2008-04-12 01:26:58.000000000 +0200
+++ grass-preview-6.3.0/lib/init/grass-xterm-wrapper	2008-04-21 14:09:45.000000000 +0200
@@ -3,7 +3,7 @@
 
 if [ -z "$GRASS_XTERM" ] ; then
    # find a working xterm or xterm compatible executable
-   TERM_TRY="xterm rxvt x-terminal-emulator"
+   TERM_TRY="x-terminal-emulator rxvt xterm"
    for TERM_PROG in $TERM_TRY ; do
       if [ -x "`which $TERM_PROG`" ] ; then
          GRASS_XTERM="$TERM_PROG"
